site/content/en/docs/tasks/manage/enforce_job_management/manage_jobs_without_queue_name.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Setup manageJobsWithoutQueueName"
+date: 2024-06-28
+weight: 10
+description: >
+  Using manageJobsWithoutQueueName and mangedJobsNamespaceSelector to prevent admission of Workloads without assigned LocalQueues.
+---
+
+This page describes how to configure Kueue to ensure that all Workloads submitted in namespaces
+intended for use by _batch users_ will be managed by Kueue even if they lack a local-queue label.
```

Comment:
```suggestion
intended for use by _batch users_ will be managed by Kueue even if they lack the queue-name label.
```
maybe we also can give a useful link

Reply:
I changed to use the actual label not a shorthand.

Reply:
fine